tools/bin/publish_docker.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+# List of directories without "airbyte-" prefix.
+projectDir=( 
+  "workers"
+  "cli"
+  "webapp"
+  "server"
+  "temporal"
+  "container-orchestrator"
+  "config/init"
+  "bootloader"
+  "metrics/reporter"
+  "db/lib"
+  "scheduler/app"
+)
+
+# Set default values to required vars. If set in env, values will be taken from there.
+JDK_VERSION=${JDK_VERSION:-17.0.1}
+ALPINE_IMAGE=${ALPINE_IMAGE:-alpine:3.14}
+POSTGRES_IMAGE=${POSTGRES_IMAGE:-postgres:13-alpine}
+
+# Iterate over all directories in list to build one by one.
+# metrics-reporter are exception due to wrong artifact naming 
+for workdir in "${projectDir[@]}"
+  do
+    if [ $workdir = "metrics/reporter" ]; then
+      artifactName="metrics-reporter" 
+    else
+      artifactName=${workdir%/*}
+    fi
+    docker buildx create --use --name $artifactName &&      \
```

Comment:
nit: what is the purpose of the `&&` here? because we `set -e` above `&&` will be seen as a test and therefore a valid command flow even with an exit 1.  This will cause lines 33 to ignore `set -e` (as opposed to exiting like we usually expect `set -e` to do)

This is a shell oddity so I made a test script to test my assumption.  I just want to make sure you intend to continue execution if line 33 exits 1

test script
```
runbooks $ cat test.sh                                                                                    
set -e

invalid_command && echo cool
echo i still run

actual_error
echo i dont run
runbooks $ bash test.sh                                                                                      
test.sh: line 3: invalid_command: command not found
i still run
test.sh: line 6: actual_error: command not found
```

Reply:
In this case it executes in a loop. It should work as I expected because if any command will return !=0, script will fail and execution will brake in current iteration.

Reply:
And one more thing. 
Line `docker buildx rm $artifactName` should be executed anyway, because if you'll have created context with the same name you'll have to remove it manually.